odoo/addons/base/models/res_partner.py

```diff
@@ -869,6 +869,12 @@ def write(self, vals):
         if 'is_company' in vals and not self.env.su and self.env.user.has_group('base.group_partner_manager'):
             result = super(ResPartner, self.sudo()).write({'is_company': vals.get('is_company')})
             del vals['is_company']
+
+        if vals.get('bank_ids'):
+            for bank_id in vals['bank_ids']:
+                bank = self.env['res.partner.bank'].browse(bank_id[1])
+                bank.write({'active': True})
```

Comment:
same here

Reply:
Also we could do this
""""
banks = self.env['res.partner.bank'].browse([bank_id[1] for bank_id in vals['bank_ids'])
bank.action_unarchive()
"""
So that we do in action in batch :smile: